models/classes/validator/PropertyChangedValidator.php

```diff
@@ -33,6 +34,22 @@ public function isPropertyChanged(core_kernel_classes_Property $property, OldPro
             return true;
         }
 
+        $this->isPropertyTypeChanged($property, $oldProperty);
+        
+        $this->isRangeChanged($property, $oldProperty);
+
+        return false;
+    }
+    public function isRangeChanged(core_kernel_classes_Property $property, OldProperty $oldProperty): bool
+    {
+        $propertyRange = $property->getRange() ? (string)$property->getRange()->getUri() : null;
+        if ($propertyRange !== $oldProperty->getrangeUri()) {
+            return true;
+        }
+        return false;
```

Comment:
Can do it in a single expression

```suggestion
        return ($property->getRange() ? (string)$property->getRange()->getUri() : null) !== $oldProperty->getrangeUri();
```